ViewModels/SignInViewModel.cs

```diff
@@ -75,13 +77,25 @@ private bool CanExecuteCommand()
 
         private void SignInInplementation(object obj)
         {
+           
+            if(_login=="admin"&& StationManager.MainPassword.Password == "admin" 
+                && StationManager.DataStorage.DirectorExists())
```

Comment:
Wait a sec, if director doesn't exists, we should allow 'admin' access

Reply:
well, sorry, wrong name of method